src/run.jl

```diff
@@ -309,18 +309,9 @@ function run_simulation(mesh, solver, time_parameters, time_integration_function
     end
 
     # Check steady-state integration residual
-    if solver.equations isa HyperbolicDiffusionEquations2D
-      if maximum(abs, view(solver.elements.u_t, 1, :, :, :)) <= solver.equations.resid_tol
-        println()
-        println("-"^80)
-        println("  Steady state tolerance of ",solver.equations.resid_tol," reached at time ",time)
-        println("-"^80)
-        println()
-        finalstep = true
-      end
-    end
-    if solver.equations isa HyperbolicDiffusionEquations3D
-      if maximum(abs, view(solver.elements.u_t, 1, :, :, :, :)) <= solver.equations.resid_tol
+    if solver.equations isa HyperbolicDiffusionEquations2D ||
+      solver.equations isa HyperbolicDiffusionEquations3D
```

Comment:
```suggestion
    if solver.equations isa AbstractHyperbolicDiffusionEquations
```
As discussed in the main thread